packages/stripe_platform_interface/lib/src/method_channel_stripe.dart

```diff
@@ -228,18 +233,20 @@ class MethodChannelStripe extends StripePlatform {
     }
   }
 
-  PaymentSheetResult _parsePaymentSheetResult(Map<String, dynamic>? result) {
+  void _parsePaymentSheetResult(Map<String, dynamic>? result) {
     if (result != null) {
       if (result.isEmpty) {
-        return const PaymentSheetResult.success();
+        return;
       } else {
         if (result['error'] != null) {
           //workaround for tojson in sumtypes
           result['runtimeType'] = 'failed';
-          return PaymentSheetResult.fromJson(result);
+          return ResultParser<void>(parseJson: (_) => {})
```

Comment:
this seems weird - why parse at all then?

Reply:
I agree will fix it